packages/dds/map/src/mapKernel.ts

```diff
@@ -184,7 +184,7 @@ export class MapKernel {
 		private readonly isAttached: () => boolean,
 		private readonly eventEmitter: TypedEventEmitter<ISharedMapEvents>,
 	) {
-		this.localValueMaker = new LocalValueMaker(serializer);
+		this.localValueMaker = new LocalValueMaker();
```

Comment:
Why moving this from constructor param? I would expect the opposite - that we would get rid of `private readonly serializer` from `MapKernel`.  Looking at its usages, it's used in `set`, I think that's a `makeSerializable` that you want to remove.  Take a look, I could be missing something.

Reply:
That `makeSerializable` is actually load-bearing and without it several hundred tests fail

Reply:
That makes me kind of worried.  Taking a quick look at the test failures.